### ISSUE TITLE
Multi-line comments aren't handled correctly

### ISSUE THREAD

**Issue by mark0n** [cmake-lint/cmake-lint#19]
cmakelint reports a false-positive for the following CMakeLists.txt:
```
#[[ this is a multi-line comment with a
copyright (c) notice
]]
```
Output:
```
test.cmake:2: Extra spaces between 'copyright' and its () [whitespace/extra]
Total Errors: 1
```
It seems like cmakelint is not handling [CMake's bracket comments](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-comment) at all.

**Comment by tkruse** [comment]
Thanks for the report. I agree that this is not implemented. cmakelint processes files line-by-line, so it's a bit annoying to implement probably. Still I would accept PRs.
